go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/skuid/spec
 
-go 1.24
+go 1.24.0
+
+toolchain go1.24.4
```

Comment:
It is not common on our projects to have this, I usually avoid to do this, is this expected?

Reply:
No, I'm pretty sure GoLand "helpfully" added this for me. I'll revert it.